lib/src/numeric_matchers.dart

```diff
@@ -63,7 +63,7 @@ class _InRange extends FeatureMatcher<num> {
   const _InRange(
       this._low, this._high, this._lowMatchValue, this._highMatchValue);
 
-  bool typedMatches(value, Map matchState) {
+  bool typedMatches(num value, Map matchState) {
```

Comment:
Can we change the signature on the `typedMatches` method above as well? Also is this not a breaking change or is this not exposed directly?

Reply:
oh, I think I shouldn't have included this. I'm pretty sure we're getting it explicitly.